pywbem/cim_operations.py

```diff
@@ -1944,6 +2269,9 @@ class are to be included in the returned instances, as follows:
                 MaxObjectCount=MaxObjectCount,
                 **extra)
 
+        if MaxObjectCount is not None and MaxObjectCount < 0:
+            raise ValueError('MaxObjectCount=%s. Invalid lt 0' % MaxObjectCount)
```

Comment:
There were multiples and fixed them all.  Note that since we have to add this test to all open/pulls I will make it a common function but right now concentrating on the Iter...